lib/tapjoy/autoscaling_bootstrap.rb

```diff
@@ -168,6 +171,7 @@ def configure_environment(filename, env=nil)
 
         new_config = defaults_hash.merge!(env_hash).merge(facet_hash)
         new_config[:config_dir] = config_dir
+        new_config[:instance_ids] = opts[:instance_ids] if opts.key?(:instance_ids)
```

Comment:
Are you recommending `new_config[:instance_ids] = opts[:instance_ids] if opts[:instance_ids]`?